src/lib/model/OutboundTransfersModel.js

```diff
@@ -166,15 +170,22 @@ class OutboundTransfersModel {
      * then waits for a notification from the cache that the payee has been resolved.
      */
     async _resolvePayee() {
+        const histTimerEnd = Metrics.getHistogram(
+            'outbound_resolve_payee',
```

Comment:
Same comment applies here for metric re-use based on map and the rest of the file.

Reply:
decided not to go with this option as name in route map requires the path to access